### PR TITLE
Add hug command

### LINKED ISSUE #46: Hug command
Give someone hug.

Usage:
`;hug @someone`

Result
`:peppoHugger: @someone`

### PR DESCRIPTION
**Pull request by albertmoravec**
Adds hug command.

I went with authorized-only permission to prevent unauthorized users from spamming mentions.

This needs to be tested remotely as I can't get it to run locally at the moment.

Closes #46.

### CODE REVIEW

**Review by tenhobi — CHANGES_REQUESTED** [review]

File: src/HonzaBotner.Discord.Services/Commands/HugCommand.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+{
+    public class HugCommand : BaseCommand
+    {
+        public const string ChatCommand = "hug";
+        // ;hug <user-mention>
+
+        protected override bool CanBotExecute => false;
+        protected override CommandPermission RequiredPermission => CommandPermission.Authorized;
+
+        protected const string HugEmoteName = "peepoHugger";
```
[the remark this answers is not in the document]
Comment (ostorc) [reply src/HonzaBotner.Discord.Services/Commands/HugCommand.cs]: Určitě ano

[the remark this answers is not in the document]
Comment (albertmoravec) [reply src/HonzaBotner.Discord.Services/Commands/HugCommand.cs]: Hotovo (snad)

File: src/HonzaBotner/appsettings.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
         "GuildId": 721264747466719285
     },
     "CommonCommandOptions": {
-        "MuteRoleId": 750276752663904306
+        "MuteRoleId": 750276752663904306,
+        "HugEmoteName": "peepoHugger"
```
Comment (tenhobi) [inline src/HonzaBotner/appsettings.json:15]: Rather use id? `772088050615582760`

**Review by tenhobi — CHANGES_REQUESTED** [review]

File: src/HonzaBotner/appsettings.json (hunk 2 above)
Comment (albertmoravec) [reply src/HonzaBotner/appsettings.json:15]: This should be fixed in #57 rather... and set on deployment env anyway.

**Pull request merged**